classes/zombie.rb

```diff
@@ -1,39 +1,111 @@
 class Zombie
-  attr_accessor :name, :username, :password, :image, :location, :bio, :created_at, :tweets, :next_tweet_id, :stalkers, :prey
+  attr_accessor :name, :username, :password, :image,
+                        :location, :bio, :created_at, :tweets, :stalkers,
+                        :prey, :retweet # :logged_in
 
+  # Class level array of Zombies, used for storing & searching in a few places
+  @@instances = []
```

Comment:
I recommend being consistent with how you name this. In `Tweet`, you have `@@all_tweets`.